e2e/const.go

```diff
@@ -1,4 +1,4 @@
-package e2e_test
+package e2e
```

Comment:
This was annoying me too. Glad you fixed it!

Reply:
For some reason it made Goland going nuts. Probably a bug, but I couldn't find other solution.